bigtable/test_bigtable.py

```diff
@@ -0,0 +1,111 @@
+"""
+This module provides testing functionality of the BigTable Init Action.
+
+Test logic:
+1. Create test table and fill it with some data by injecting commands into hbase shell.
+2. Validate from local station that BigTable has test table created with right data.
+
+Note:
+    Test REQUIRES cbt tool installed which provides CLI access to BigTable instances.
+    See: https://cloud.google.com/bigtable/docs/cbt-overview
+"""
+import unittest
+import random
+import os
+
+from parameterized import parameterized
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class BigTableTestCase(DataprocTestCase):
+    COMPONENT = 'bigtable'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/bigtable/bigtable.sh'
+    TEST_SCRIPT_FILE_NAME = "run_hbase_commands.py"
+    METADATA = None
```

Comment:
Nit: if these aren't constants they should be initialized in `__init__` as `self.lowercasevariable = None`

Reply:
ok